rero_ils/modules/operation_logs/jsonschemas/operation_logs/operation_log-v0.0.1.json

```diff
@@ -78,6 +78,187 @@
           "pattern": "^https://ils.rero.ch/api/organisations/.+?$"
         }
       }
+    },
+    "loan": {
+      "title": "Loan",
+      "description": "Loan specific data",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "override_flag": {
+          "title": "Override flag",
+          "type": "boolean",
+          "default": false
+        },
+        "transaction_channel": {
+          "title": "Transaction channel",
+          "type": "string",
+          "enum": [
+            "ui",
+            "system",
+            "sip2"
+          ]
+        },
+        "transaction_user_name": {
+          "title": "Transaction user name",
+          "type": "string",
+          "minLength": 1
+        },
+        "transaction_location": {
```

Comment:
It seems overkill to have an object with only one property. Why not `transation_location_name`?